common/swagger/swagger.yaml

```diff
@@ -141,9 +164,70 @@ definitions:
     type: array
     items:
       type: string
+
+  ContributionRequest:
+    type: object
+    required:
+      - requestShapleyValueType
+    properties:
+      requestShapleyValueType:
+        $ref: '#/definitions/ShapleyType'
+        description: >
+          The string to say what type of Shap values are needed.
+          `ORIGINAL` implies Shap values of original features are requested,
+          `TRANSFORMED` implies that Shap values of transformed features are requested.
+        type: string
+      fields:
+        description: >
+          An array holding the names of fields in the order of appearance in the `rows` property. The length of `fields`
+          has to match length of each row in `rows`. No duplicates are allowed.
+        type: array
+        items:
+          type: string
+      rows:
+        description: >
+          An array of rows consisting the actual input data for scoring, one scoring request per row.
+        type: array
+        items:
+          $ref: '#/definitions/Row'
+  ContributionResponse:
+    type: object
+    properties:
+      features:
+        description: >
+          An array holding the names of fields in the order of appearance in the rows of the  `contributions` property.
+        type: array
+        items:
+          type: string
+      contributionByOutputGroup:
+        description: >
+          An array of rows consisting of the shapley contributions output corresponding to an output group.
+        type: array
+        items:
+          $ref: '#/definitions/ContributionByOutputGroup'
+  ContributionByOutputGroup:
```

Comment:
Could it make sense to adjust this name just a bit more?  Other fields are in `camelCase`, and variable-sized/array fields are in plural form.

IMHO:  `*OutputGroup` feels out of place.  Perhaps `contributions` could have used here, but then of course the nested `contributions` would have to take on another name.  Or `contributionGroups`, or something else.  Not sure, TBH - just sharing thoughts.

Reply:
`ContributionByOutputGroup` is not the field name. It is the definition. The field name is a camelCase `contributionByOutputGroup`, but I get your point, I can name this as `contributionGroups` if that is ok